[2023-10-20T11:18:57.539+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple.pokelist manual__2023-10-20T01:47:50.286271+00:00 [queued]>
[2023-10-20T11:18:57.547+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple.pokelist manual__2023-10-20T01:47:50.286271+00:00 [queued]>
[2023-10-20T11:18:57.548+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-10-20T11:18:57.560+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): pokelist> on 2023-10-20 01:47:50.286271+00:00
[2023-10-20T11:18:57.567+0000] {standard_task_runner.py:57} INFO - Started process 58 to run task
[2023-10-20T11:18:57.573+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple', 'pokelist', 'manual__2023-10-20T01:47:50.286271+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/pokelist.py', '--cfg-path', '/tmp/tmp44wc21gz']
[2023-10-20T11:18:57.577+0000] {standard_task_runner.py:85} INFO - Job 46: Subtask pokelist
[2023-10-20T11:18:57.633+0000] {task_command.py:416} INFO - Running <TaskInstance: simple.pokelist manual__2023-10-20T01:47:50.286271+00:00 [running]> on host 90af428214e9
[2023-10-20T11:18:57.727+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple' AIRFLOW_CTX_TASK_ID='pokelist' AIRFLOW_CTX_EXECUTION_DATE='2023-10-20T01:47:50.286271+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-20T01:47:50.286271+00:00'
[2023-10-20T11:18:57.730+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-20T11:18:57.730+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/scripts/pokelist.py']
[2023-10-20T11:18:57.750+0000] {subprocess.py:86} INFO - Output:
[2023-10-20T11:18:57.858+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-10-20T11:18:57.859+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/pokelist.py", line 2, in <module>
[2023-10-20T11:18:57.860+0000] {subprocess.py:93} INFO -     from pyspark.sql import SparkSession
[2023-10-20T11:18:57.860+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'pyspark'
[2023-10-20T11:18:57.861+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-10-20T11:18:57.883+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-10-20T11:18:57.888+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=simple, task_id=pokelist, execution_date=20231020T014750, start_date=20231020T111857, end_date=20231020T111857
[2023-10-20T11:18:57.908+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 46 for task pokelist (Bash command failed. The command returned a non-zero exit code 1.; 58)
[2023-10-20T11:18:57.943+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-20T11:18:57.973+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
